crates/vault-registry/src/slash.rs

```diff
@@ -29,6 +29,7 @@ pub enum SlashingError {
     ArithmeticUnderflow,
     ArithmeticOverflow,
     InsufficientFunds,
+    VaultNotFound,
```

Comment:
This can be removed, looks to be unused

Reply:
Ah okay, so the advantage of using `E: From<SlashingError>` on the trait is that the `Error` type can be anything as long as we can cast `SlashingError` to it. This means we don't need to add additional variants here if you implement `RichNominator` with the pallet's `Error` type.

Reply:
Better link to usage: https://github.com/interlay/btc-parachain/blob/dan/nomination-pull/crates/nomination/src/types.rs#L381